site/source/docs/building_from_source/building_emscripten_from_source_using_the_sdk.rst

```diff
@@ -116,7 +116,7 @@ After installing the main repositories:
 
 		::
 
-			cmake .. -DCMAKE_BUILD_TYPE=Release -DLLVM_TARGETS_TO_BUILD="host;JSBackend" -DLLVM_INCLUDE_EXAMPLES=OFF -DLLVM_INCLUDE_TESTS=OFF -DCLANG_INCLUDE_TESTS=OFF
+			cmake ../src -DCMAKE_BUILD_TYPE=Release -DLLVM_TARGETS_TO_BUILD="host;JSBackend" -DLLVM_INCLUDE_EXAMPLES=OFF -DLLVM_INCLUDE_TESTS=OFF -DCLANG_INCLUDE_TESTS=OFF
```

Comment:
@juj is this necessary for the emsdk, which creates a `src/` directory?

Reply:
Oh hmm - this is quite awkward. This documentation is under section `Building Emscripten using your own fork`, which documents the set of steps that one would use to replace each of Emscripten, Emscripten-fastcomp and Emscripten-fastcomp-clang with one's own remote forks, and rebuild each. What makes this awkward is that this feels like something people would very rarely want to do: instead what practically everyone would want is to use one's own fork of Emscripten, but reuse kripken/emscripten-fastcomp and kripken/emscripten-fastcomp-clang unchanged.

If one is using emsdk to bootstrap Emscripten+emscripten-fastcomp+emscripten-fastcomp-clang and then wants to replace each of these with one's own fork, then yes, one would go into the build directory (e.g. `C:\emsdk\clang\fastcomp\build_incoming_vs2017_64`), and issue `cmake ../src` (to target `C:\emsdk\clang\fastcomp\src` which is where the sources are cloned to)

But generally if one wants to just use one's own fork of Emscripten, they'd do

```bash
git clone https://github.com/juj/emsdk.git
cd emsdk
emsdk install sdk-incoming-64bit
emsdk activate sdk-incoming-64bit
source ./emsdk_env.sh
cd emscripten/incoming
git remote add my_own_fork https://github.com/my_own_fork/emscripten.git
git fetch my_own_fork
git checkout my_branch
```

and they'd be all set, i.e. this scheme reuses kripken/emscripten-fastcomp and kripken/emscripten-fastcomp-clang, but moves the working tree in emscripten/incoming over to the custom fork.

Reply:
Yes, I found weird to have to fork fastcomp and fastcomp-clang also, even if I only want to work on emscripten...